Tasks/duplicate.js

```diff
@@ -1,14 +1,12 @@
+'use strict';
 // Return an array without duplicates
 
-duplicate = (value, N) => {
-  if (N <= 0) return [];
-  else {
-    res = [];
-    for (let i = 0; i < N; i++) {
-      res[i] = value;
-    }
-    return res;
+const duplicate = (value, N) => {
```

Comment:
Use `camelCase` so `n` or `count`
```suggestion
const duplicate = (value, n) => {
```